src/main/java/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointTypeUtils.java

```diff
@@ -0,0 +1,246 @@
+          Quantities.getQuantity(3.6, StandardUnits.S_RATED),
+          AC,
+          new HashSet<>(Arrays.asList("bluehousehold", "bhs", "schuko-camping")));
+
+  public static final ChargingPointType Cee16ASocket =
+      new ChargingPointType(
+          "Cee16ASocket",
+          Quantities.getQuantity(11, StandardUnits.S_RATED),
+          AC,
+          Collections.singleton("cee16"));
+
+  public static final ChargingPointType Cee32ASocket =
+      new ChargingPointType(
+          "Cee32ASocket",
+          Quantities.getQuantity(22, StandardUnits.S_RATED),
+          AC,
+          Collections.singleton("cee32"));
+
+  public static final ChargingPointType Cee63ASocket =
+      new ChargingPointType(
+          "Cee63ASocket",
+          Quantities.getQuantity(43, StandardUnits.S_RATED),
+          AC,
+          Collections.singleton("cee63"));
+
+  public static final ChargingPointType ChargingStationType1 =
+      new ChargingPointType(
+          "ChargingStationType1",
+          Quantities.getQuantity(7.2, StandardUnits.S_RATED),
+          AC,
+          new HashSet<>(Arrays.asList("cst1", "stationtype1", "cstype1")));
+
+  public static final ChargingPointType ChargingStationType2 =
+      new ChargingPointType(
+          "ChargingStationType2",
+          Quantities.getQuantity(43, StandardUnits.S_RATED),
+          AC,
+          new HashSet<>(Arrays.asList("cst2", "stationtype2", "cstype2")));
+
+  public static final ChargingPointType ChargingStationCcsComboType1 =
+      new ChargingPointType(
+          "ChargingStationCcsComboType1",
+          Quantities.getQuantity(11, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("csccs1", "csccscombo1")));
+
+  public static final ChargingPointType ChargingStationCcsComboType2 =
+      new ChargingPointType(
+          "ChargingStationCcsComboType2",
+          Quantities.getQuantity(50, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("csccs2", "csccscombo2")));
+
+  public static final ChargingPointType TeslaSuperChargerV1 =
+      new ChargingPointType(
+          "TeslaSuperChargerV1",
+          Quantities.getQuantity(135, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("tesla1", "teslav1", "supercharger1", "supercharger")));
+
+  public static final ChargingPointType TeslaSuperChargerV2 =
+      new ChargingPointType(
+          "TeslaSuperChargerV2",
+          Quantities.getQuantity(150, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("tesla2", "teslav2", "supercharger2")));
+
+  public static final ChargingPointType TeslaSuperChargerV3 =
+      new ChargingPointType(
+          "TeslaSuperChargerV3",
+          Quantities.getQuantity(250, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("tesla3", "teslav3", "supercharger3")));
+
+  protected static final Map<String, ChargingPointType> commonChargingPointTypes =
```

Comment:
Pls add JavaDoc. I think, it just concats all available ids and synonymous ids and maps them to the actual charging point types?